src/components/Login/LoginContainer.tsx

```diff
@@ -133,12 +124,17 @@ export const InitialisationContainer: FunctionComponent<NavigationProps> = ({
     });
   };
 
+  const setState = useState()[1];
   const handleRequestOTP = async (
     fullNumber = mobileNumber
   ): Promise<boolean> => {
     try {
       if (!(await getResendConfirmationIfNeeded())) return false;
-      const response = await requestOTP(fullNumber, codeKey, endpointTemp);
+      const response = await requestOTP(
+        fullNumber,
+        tempAuthCredentials?.operatorToken ?? "",
+        tempAuthCredentials?.endpoint ?? ""
```

Comment:
May be a noob question, but why is it possible for operatorToken and endpoint to be empty? Is this for dev environment?

Reply:
TypeScript will complain that `requestOTP` requires non-undefined params if we don't set the default to empty string. If instead we update the signature of `requestOTP` to take in undefined variables, then we're indicating that `requestOTP` has optional params, which should not be the case.

If you're wondering why `tempAuthCredentials` can be undefined, it's because we don't initialise it to any particular default value. The call to `handleRequestOTP` will only be called after `operatorToken` and `endpoint` have been defined.

How do you think this can be better approached?

Reply:
I think it's fine! I was just curious as to why this was the case. If the call is only made after they have been defined then it is fine.